### ISSUE TITLE
Update coefficients for SPECpower Q3 2021

### ISSUE THREAD

**Issue by davidmytton** [cloud-carbon-footprint/cloud-carbon-footprint#383]
Update [the coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients) to include the results of the Q3 2021 tests at https://www.spec.org/power_ssj2008/results/res2021q3/

**Comment by ericksod** [comment]
Hi David! Thanks for adding this. Were you thinking of picking it up yourself or should we put it in the Ready To Do?

**Comment by davidmytton** [comment]
Yeah I was planning to take this one. Can you assign to me please if nobody else is going to do it over the next few days?

**Comment by davidmytton** [comment]
I've just opened https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/pull/5 which updates the SPECpower dataset. There are no changes to the coefficients from this update because all servers are in the Ice Lake arch which isn't currently covered. Something to look into after this issue.

**Comment by ccasher** [comment]
Hi @davidmytton, upon merging the PR into the coefficients repo, we noticed that there were actually quite a few coefficients that changed. Was this intended, or were you thinking there would be no changes since the servers were all in Ice Lake? You can see some of the updates [here](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/coefficients.ipynb)

Also, would you mind reopening [this PR](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/pull/411)? We decided to revert the changes until we get this resolved.

**Comment by davidmytton** [comment]
The merge of https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/pull/5 should have no coefficients changed because they were all in Ice Lake. However, the values in the notebook don't match the values in this repo. During the course of https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/issues/331 we found some discrepancies and so there was a pending task to make sure they were both the same.

I updated the data to match SPECpower Q3 2021 to make sure that the latest data was there, then https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/pull/411 ensures the values match.
